### PR TITLE
chore: Fixed typos and removed not referred private functions in google client SDK

### PR DESCRIPTION
**Pull request by toshitanian**
**Description of your changes:**

Fixed several typos and removed not referred private functions in google client SDK

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: sdk/python/kfp/v2/google/client/client.py [hunk 1]
```diff
@@ -361,7 +361,7 @@ def create_schedule_from_job_spec(
     be paused/resumed and deleted.
 
     To make the system work, this function also creates a Google Cloud Function
-    which acts as an intermediare between the Scheduler and Pipelines. A single
+    which acts as an intermediary between the Scheduler and Pipelines. A single
```
Comment (toshitanian) [inline sdk/python/kfp/v2/google/client/client.py:364]: `intermediare` seems to be French

File: sdk/python/kfp/v2/google/client/schedule.py [hunk 2]
```diff
@@ -33,7 +33,6 @@
 _PROXY_FUNCTION_FILENAME = '_cloud_function_templated_http_request.py'
 
 _CAIPP_DEFAULT_ENDPOINT = 'us-central1-aiplatform.googleapis.com'
-_CAIPP_SERVICE_NAME = 'aiplatform'
```
Comment (toshitanian) [inline sdk/python/kfp/v2/google/client/schedule.py:36]: Not referred event outside the file.

File: sdk/python/kfp/v2/google/client/schedule.py [hunk 3]
```diff
@@ -320,18 +319,6 @@ def _create_or_get_cloud_function(
   return function_get_response
 
 
-def _initialize_and_get_endpoint(
```
Comment (toshitanian) [inline sdk/python/kfp/v2/google/client/schedule.py:323]: Not referred event outside the file.

**Comment by chensun** [comment]
/ok-to-test
/lgtm
/approve

Thanks, @toshitanian!

**Pull request merged**